### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
+To find a particular layout item by its name, call the control's [FindItemOrGroupByName](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFormLayout.FindItemOrGroupByName(System.String)) method and pass the item's `Name` property value as a parameter.
```
Comment (arminal) [inline Readme.md]:
```suggestion
To find a layout item by its name, call the control's [FindItemOrGroupByName](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFormLayout.FindItemOrGroupByName(System.String)) method and pass the item's `Name` property value as a parameter.
```

File: Readme.md [hunk 2]
```diff
@@ -1,22 +1,53 @@
...
 
-<br/>
+```csharp
+LayoutItemBase baseItem = layout.FindItemOrGroupByName(ASPxComboBox1.Value.ToString()) as LayoutItemBase;
+```
+
+To get a nested control inside a layout item, use the [LayoutItem.Controls](https://docs.devexpress.com/AspNet/DevExpress.Web.LayoutItem.Controls) property. The code sample below traverses through the collection of nested controls and assigns new values to the controls.
```
Comment (arminal) [inline Readme.md]: Question about phrasing: to get a nested control inside a layout item? to get something inside something? That phrasing is odd. What do you mean by "get" here?

File: findItem.png
Comment (arminal) [inline findItem.png:1]: The text in this image: "This item has two inner controls, so the value should be changed in both"

Let's rephrase: "This item has two inner controls; change the value in both of them." OR "...assign a value to both of them." Which ever is technically correct.

File: Readme.md (hunk 2 above)
Comment (elenapeskova) [reply Readme.md]: Here I describe how to access a control placed in a layout item. So can we change it to the following:
```suggestion
To access a control placed in a layout item, use the [LayoutItem.Controls](https://docs.devexpress.com/AspNet/DevExpress.Web.LayoutItem.Controls) property. The code sample below traverses through the collection of nested controls and assigns new values to the controls.
```

File: findItem.png
Comment (elenapeskova) [reply findItem.png:1]: Updated

File: Readme.md (hunk 2 above)
Comment (arminal) [reply Readme.md]: Great! Thank you.

**Pull request merged**